src/update_parameters.jl

```diff
@@ -190,13 +192,122 @@ function update_parameter_in_quadratic_objective_pp!(model::ParametricOptimizer)
     end
 end
 
+# Conic
+function update_parameter_in_conic_constraints_nn!(model::ParametricOptimizer)
+    conic_constraint_cache_nn_inner = model.conic_constraint_cache_nn[MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives]
+
+    if !isempty(conic_constraint_cache_nn_inner)
+        update_parameter_in_conic_constraints_nn!(model.optimizer, model.parameters, model.updated_parameters, conic_constraint_cache_nn_inner)
+    end
+
+    return model
+end
+
+function update_parameter_in_conic_constraints_nn!(
+    optimizer::OT, 
+    parameters::Dict{MOI.VariableIndex, T}, 
+    updated_parameters::Dict{MOI.VariableIndex, T}, 
+    conic_constraint_cache_nn_inner::DD.WithType{F, S}) where {OT, T, F, S}
+
+    for (ci, param_array) in conic_constraint_cache_nn_inner
+        update_parameter_in_conic_constraints_nn!(optimizer, ci, param_array, parameters, updated_parameters)
+    end
+
+    return optimizer
+end
+
+function update_parameter_in_conic_constraints_nn!(
+                                optimizer::OT,
+                                ci::CI, 
+                                param_array::Vector{MOI.VectorAffineTerm{T}},
+                                parameters::Dict{MOI.VariableIndex, T}, 
+                                updated_parameters::Dict{MOI.VariableIndex, T},
+                            ) where {OT, T, CI}
+
+    # TODO
+    # I suppose this get works fine here but I'm not sure
+    cf = MOI.get(optimizer, MOI.ConstraintFunction(), ci)
+
+    n_dims = length(cf.constants)
+    param_constants = zeros(T, n_dims)
+
+    for term in param_array
+        vi = term.scalar_term.variable_index
+
+        if haskey(updated_parameters, vi) # TODO This haskey can be slow
+            param_constants[term.output_index] = term.scalar_term.coefficient * (updated_parameters[vi] - parameters[vi])
+        end
+    end
+
+    if param_constants != zeros(T, n_dims)
+        MOI.modify(optimizer, ci, MOI.VectorConstantChange(cf.constants + param_constants))
+    end
+
+    return ci
+end
```

Comment:
no need for separate code for each vector set.
nothing in this function depends on the set type... so this would work for any vector set.